Deployment/default.ps1

```diff
@@ -60,3 +60,15 @@ Task CreateRap -Depends RestoreBuildTools, PackageBuild {
 	}
 	Copy-Item $scriptrunner_rap -Destination "$Packages\$Version\Milyli.ScriptRunner-$Version.rap" -Force | Out-Null
 }
+
+Task CreateNugetPkg -Depends PackageBuild {
```

Comment:
This Task depends on the `InstallNuget` Task and that needs to be added to the -Depends list.

 - [ ] Add the task dependency

Reply:
My understanding is that the dependencies cascade; the `PackageBuild` task depends on `RestorePackages` which depends on `InstallNuget`, so we're already covered.

Reply:
They cascade for now, but if the PackageBuild task is later changed to not depend on a Task that installs nuget this Task will break without warning. It's probably best to be as explicit as possible, especially since the nuget Task has a precondition and isn't expensive.

Reply:
This still needs the InstallNuget task dependency made explicit

 - [ ] Add the task dependency